### PR TITLE
content: initial tinlake doc

### PR DESCRIPTION
**Pull request by manuelpolzhofer**
please take a final review on the intial tinlake documentation.

### CODE REVIEW

File: docs/tinlake/overview/introduction.md [hunk 1]
```diff
@@ -7,46 +7,18 @@ redirect_from:
...
- | Collateral | The collateral value token. |
- | Appraiser| Appraiser is a module that can be plugged in that returns the value of the collateral. | 
- | Reception | Wrapper contract for borrowers |
- | Desk | The "trading desk", wrapper contract for the administrator to manage the lender side |
- | MakerAdapter | Manages a CDP |
- | BackerAdapter | Uses a single account to dra
- | Deployer | Deploys all components in one transaction setting the right wards |
- 
+### Administrator
+Generally, an administrator determines what assets borrowers are allowed to bring in as collateral and how much they are allowed to borrow against the assets. This administrator could be a single centralized entity, but could be a set of automated smart contracts automatically qualifying and pricing an asset. Or, you could plug in a DAO that takes control of the Tinlake deployment.
```
Comment (denniswell) [inline docs/tinlake/overview/introduction.md]: ... This administrator could be a single centralized entity, a set of smart contracts automatically qualifying and pricing an asset or a decentralised autonomous organisation (DAO) that manages the Tinlake deployment.

File: docs/tinlake/overview/patterns.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+redirect_from:
+  - /tinlake/
+---
+
+### Small modular contracts
+The Tinlake contracts use the ward pattern for authorizing internal contracts to modify system parameters. To modularize and simplify the code base, independent functions are put into different contracts that can be changed out. 
+
+## Contract Call Authorization Scheme
+Most contracts have a very simple authorization mechanism for separating public calls that anyone can do and system internal calls. This pattern was established by DappHub/MakerDAO. Instead of more fine grained complex access control within the contract, permissions are binary. Limiting what actions callers can do is not necessary if the caller is a smart contract, as this can be enforced on the caller side by simply writing the contract such that it can't do anything it shouldn't do.
+
```
Comment (denniswell) [inline docs/tinlake/overview/patterns.md]: Perhaps include link to DappHub/MakerDAO?

File: docs/tinlake/overview/liquidation.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+category: 5 . Further Information
+redirect_from:
+  - /tinlake/
+---
+
+The total amount of CVT minted by the Tinlake contracts always represents the value of the entire pool. If a new loan is borrowed by locking an NFT, new CVT are minted to represent the pool value increase.  In case of repayment, the CVT related to the loan are burned. The value of a pool increases or decreases not only by borrowing new loans or repaying existing ones. Individual NFTs could also increase or decrease in value. For example, an invoice NFT would decrease in value if an invoice is overdue for couple of days because it increases the risk of a default. A decrease or increase of an individual assets affects the entire pool.
+
+### Liquidation by Underwriter
+If the value of an NFT within the pool drops below its required collateralization value, the underlying loan has to be repaid by the underwriter. In this scenario, the debt (principal + interest) is transferred to the landing protocol by the underwriter in exchange for CVT tokens. The underlying NFT is removed from the pool and placed in escrow, temporarily transferring ownership to the underwriter for the duration of the loan period.
+
```
Comment (denniswell) [inline docs/tinlake/overview/liquidation.md]: ... l**e**nding protocol...

Comment (denniswell) [reply docs/tinlake/overview/liquidation.md]: ... in an escrow account... (?)

File: docs/tinlake/overview/liquidation.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+### Liquidation by Lending Protocol
+Liquidation by the lending protocol should only occur in case the admin is not reacting on a value decrease of the pool which exceeds the preconfigured risk margin/threshhold. In order to protect the lenders from defaults, the admin has a high incentive to react on value decreases of the pool which bring the pool value below the defined threshhold. In the ideal case, a liquidation from the lending protocol like in CDP liquidation in Maker should never occur. In addition, each pool has a collateral factor which prevents a liquidation.
```
Comment (denniswell) [inline docs/tinlake/overview/liquidation.md]: How and by whom is the risk margin determined? Does this dependent on the asset and/or counterparty?!

Comment (denniswell) [reply docs/tinlake/overview/liquidation.md]: What is the collateral factor? Needs to be explained

File: docs/tinlake/overview/liquidation.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+Each pool can have a different collaterization ratio which determines a liquidation event.
```
Comment (denniswell) [inline docs/tinlake/overview/liquidation.md]: Same here. What is the collaterization ratio. Who determines it....

File: docs/tinlake/overview/contracts.md [hunk 6]
```diff
@@ -1,10 +1,46 @@
...
+`Admin.whitelist(address registry, uint nft, uint principal, uint appraisal, uint fee, address usr)`
+
+An admin (underwriter) can whitelist an NFT for Tinlake. The admin has to define a principal, appraisal and interest rate (APR) for the NFT. Only whitelisted NFTs can be used to receive loans.  
+
+
+#### Borrow a loan
+`Reception.borrow(uint loan, address deposit)`
+
+The owner of a whitelisted NFT can call `borrow` on the reception contract. As a first step, the ownership of the NFT is transferred to the Tinlake shelf contract. (Pre required is an `approve` in the NFT registry contract). The new locked NFT causes a value (appraisal) increase of the collateral pool. New CVT are minted to represent the increase. Afterwards, the new CVT are used as collateral in a decentralized lending protocol like Maker or Compound (depending on the adapter) to receive funding in DAI (principal amount).  The borrow transaction automatically whithdraws the received DAI to the deposit address. If borrow is successful, the loan status changes to `ongoing` and interest accrues.
+
```
Comment (denniswell) [inline docs/tinlake/overview/contracts.md]: Two spaces before ...The borrow transaction

**Comment by denniswell** [comment]
Added a few comments, but this is very solid work given the time frame.
Hats off

File: docs/tinlake/overview/deployments.md [hunk 7]
```diff
@@ -7,4 +7,5 @@ redirect_from:
   - /tinlake/
 ---
 
-contractgs
+## Tinlake Deployments
```
Comment (denniswell) [inline docs/tinlake/overview/deployments.md]: I would actually move this paragraph into the Introduction (as a paragraph, not a section

File: docs/tinlake/overview/ui.md [hunk 8]
```diff
@@ -1,10 +1,23 @@
...
 ---
 
-# Tinlake JS
\ No newline at end of file
+Tinlake UI provides a simple interface for Admins & Borrowers to interact with Tinlake Contracts. The UI supports the following set of features:
+
+**Admins’ perspective**
+- Whitelisting loans
+- Managing loans
+
```
Comment (denniswell) [inline docs/tinlake/overview/ui.md]: Should we include Screenshots of the main UI screens here?

File: docs/tinlake/overview/interest.md [hunk 9]
```diff
@@ -7,4 +7,85 @@ redirect_from:
   - /tinlake/
 ---
 
-# Interest
```
Comment (denniswell) [inline docs/tinlake/overview/interest.md]: General comment: I think, I'd have only the Examples in 
```
Code blocs
```
and have the formulas just in
`blue font`

File: docs/tinlake/overview/liquidation.md (hunk 3 above)
Comment (manuelpolzhofer) [reply docs/tinlake/overview/liquidation.md]: good catch

File: docs/tinlake/overview/liquidation.md (hunk 4 above)
Comment (denniswell) [reply docs/tinlake/overview/liquidation.md]: Should we specifically mention Maker here or just refer to it as Lending Facility?

File: docs/tinlake/overview/deployments.md (hunk 7 above)
Comment (lucasvo) [reply docs/tinlake/overview/deployments.md]: Let's leave it like that and expand on it later

**Pull request merged**